app/utils/searchUtils.js

```diff
@@ -18,9 +18,9 @@ function getFetchParamsFromFilters(filters) {
       filters.duration
     ),
     { purpose: filters.purpose === 'all' ? '' : filters.purpose },
-    { page: filters.page || 1 }
+    { page: filters.page || 1 },
+    { ...(filters.orderBy ? { order_by: filters.orderBy } : {}) },
```

Comment:
this is bad, why just `filters.orderBy ? ... {order_by: filters.orderBy) : {}`?

Reply:
This line is now refactored in new commit. The idea of using the previous way was from here : http://2ality.com/2017/04/conditional-literal-entries.html